third_party/java/java_tools/update_java_tools.sh

```diff
@@ -62,7 +62,8 @@ else
   # If no tools were specified update all of them.
   tools_to_update=(${!tool_name_to_target[*]})
   # Create the sources archive only when all the tools were updated.
-  zip -r third_party/java/java_tools/java_tools-srcs.zip src/java_tools/buildjar/* \
+  # -X doesn't save extra file attributes (e.g. file times)
+  zip -Xr third_party/java/java_tools/java_tools-srcs.zip src/java_tools/buildjar/* \
```

Comment:
As discussed offline, I don't think this is enough to remove all the times. But feel free to move and add a TODO here instead.

Reply:
Yeah. If you want to emulate what we are doing with the actual Bazel zip, you'll end up writing a Python script:

https://github.com/bazelbuild/bazel/blob/master/src/create_embedded_tools.py#L133

That would also be consistent rule of thumb of "if your shell script is more than three lines long, write it in Python" :)

I don't know by heart what kind of zip handling tools we have in this repository, but it's conceivable that something can also be jury-rigged from the code we use for ijar/singlejar.

I also agree with @meisterT that this is fine for the time being. Having a zip with time stamps beats not having a zip :)